### PR TITLE
Add a ping check with MTU packet size

### PR DESCRIPTION
**Pull request by DeepthiYV**
Still doing Debugging

- Related ticket: https://progress.opensuse.org/issues/135200
- Needles: NO
- Verification run:

### CODE REVIEW

**Comment by pdostal** [comment]
Can you please explain why do you do this?

**Comment by Martchus** [comment]
I'm wondering about the overlap with #17817. If the check is wanted in both places it would make sense to use the helper function introduced in #17817 here as well.

File: tests/network/setup_multimachine.pm [hunk 1]
```diff
@@ -39,6 +39,8 @@ sub run {
     # Set the hostname to identify both minions
     set_hostname $hostname;
 
+    # Perform ping to client from server
+    assert_script_run("ping -M do -c 1 -s 1430 10.0.2.102") if ($hostname =~ /server|master/);
```
Comment (czerw) [inline tests/network/setup_multimachine.pm:43]: @DeepthiYV PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17817 was merged, you can reuse the function which was introduced there (either with multiple sizes check or with fixed size).

**Comment by DeepthiYV** [comment]
> Can you please explain why do you do this?

This check in the Multimachine job was intended to verify network performance by testing MTU. Now that I see the tests are stable, I don't think we need this pull request anymore. So, I will close it.

**Pull request closed without merging**